models/repo_transfer.go

```diff
@@ -0,0 +1,237 @@
+package models
+
+import (
+	"fmt"
+	"os"
+
+	"code.gitea.io/gitea/modules/util"
+	"github.com/Unknwon/com"
+)
+
+// RepoStatus determines the current state of a transfer
+type RepoStatus uint8
+
+const (
+	// Rejected is a status for transfers that get cancelled by either the
+	// recipient or the user who initiated the transfer
+	Rejected RepoStatus = iota
```

Comment:
Why not `Pending` is the default value?

Reply:
When writing to the database, Pending is the default. 

I wanted 

Rejected - 0
Pending - 1
Accepted - 2

Reply:
With states it is common practice to have 0 be the default. Rejected is not false, it's a state.